talaiot/src/main/kotlin/com/cdsap/talaiot/publisher/timeline/Data.kt

```diff
@@ -0,0 +1,10 @@
+package com.malinskiy.marathon.report.debug.timeline
+
+import com.cdsap.talaiot.entities.TaskMessageState
+
+data class Data(val taskPath: String,
```

Comment:
Can we have a more descriptive name please?

Reply:
Done